core/controllers/base.py

```diff
@@ -53,7 +53,11 @@
 
 
 def _clear_login_cookies(response_headers):
-    """Clears login cookies from the given response headers."""
+    """Clears login cookies from the given response headers.
+
+    Args:
+        response_headers: dict. A set of HTTP response headers.
```

Comment:
Are you sure this will be a dict? (I see we are calling `response_headers.add_header` on line 72.)

Reply:
should it be a function?? I'm confused actually

Reply:
done